### PR TITLE
games-action/minetest{,_game}: Version bump to 0.4.14

### PR DESCRIPTION
**Pull request by damiel**
Version bump of minetest and minetest_game to 0.4.14. This commit also enables the leveldb backend by default like upstream.

I also attempted to bump to EAPI=6, portage and repoman seem to be fine with the changes done, but i would like to have a second pair of eyes balling over it.

### CODE REVIEW

File: games-action/minetest/minetest-0.4.14.ebuild [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+	# set paths
+	sed \
+		-e "s#@BINDIR@#/usr/bin#g" \
+		-e "s#@GROUP@#${PN}#g" \
+		"${FILESDIR}"/minetestserver.confd > "${T}"/minetestserver.confd || die
+}
+
+src_configure() {
+	local mycmakeargs=(
+	$(usex dedicated "-DBUILD_SERVER="$(usex server)" -DBUILD_CLIENT=OFF" "-DBUILD_SERVER="$(usex server)" -DBUILD_CLIENT=ON")
```
Comment (nvinson) [inline games-action/minetest/minetest-0.4.14.ebuild]: You could change the above two lines to
```
        local mycmakeags=(
                -DBUILD_SERVER="$(usex server)"
                $(usex dedicated "-DBUILD_CLIENT=OFF" "-DBUILD_CLIENT=ON")
        )        
```
and you would be under the 80 col restriction.

Comment (mgorny) [inline games-action/minetest/minetest-0.4.14.ebuild]: -DBUILD_CLIENT=$(usex !dedicated)

should do the same, wouldn't it?

**Comment by monsieurp** [comment]
@gentoo/proxy-maint

**Comment by idella** [comment]
All runtests fine. Only mino6 thing is line 68 need be split into 2. It reaches 88 chars.
Next time

**Pull request closed without merging**